chall03/vkuikka.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"strconv"
+	"io/ioutil"
+)
+
+func	getData(addr string) string {
```

Comment:
I would not recommend creating another function to Get Data while you already have http.Get() for that.

Plus, if one day you have to parse more information return by http.Get() into your main code you will be forced to find a way to send more parameters, and it will start to make your code less clean.